torch/export/_remove_effect_tokens_pass.py

```diff
@@ -0,0 +1,126 @@
+import operator
+from typing import List
+
+import torch
+from torch._higher_order_ops.effects import with_effects
+from .exported_program import ExportedProgram
+from .graph_signature import (
+    InputKind,
+    InputSpec,
+    OutputKind,
+    OutputSpec,
+    TensorArgument,
+)
+
+
+def remove_effect_tokens(ep: ExportedProgram) -> ExportedProgram:
```

Comment:
To make it more obviously private.
```suggestion
def _remove_effect_tokens(ep: ExportedProgram) -> ExportedProgram:
```